### ISSUE TITLE
With 3.2 update caret in normal mode is no longer visible

### ISSUE THREAD

**Issue by nikitavoloboev** [guillermooo/Six#243]
After updating Sublime Text to [3.2 version](https://www.sublimetext.com/blog/articles/sublime-text-3-point-2). My caret in normal mode started to look like this:

![2019-03-13 at 11 29](https://user-images.githubusercontent.com/6391776/54272121-532cfb80-4583-11e9-94f4-f5f32c1db0ae.png)

It is barely seen. In insert mode it is working good:

![2019-03-13 at 11 30](https://user-images.githubusercontent.com/6391776/54272143-5d4efa00-4583-11e9-9781-9b971a5389c0.png)

As you see I set `	"caret_style": "solid",` as setting if that makes a difference. Would love to receive a fix for this. 🧡

**Comment by nikitavoloboev** [comment]
Also I am not sure if I had this but I wish I could make the cursor underline when in normal mode like I can in vim with this setting:

`set guicursor=n-v-c:hor20,i-ci:ver20 " Make cursor block in insert mode and underline in normal mode`


![2019-03-13 at 11 32](https://user-images.githubusercontent.com/6391776/54272315-bd45a080-4583-11e9-89a7-4bb0f725f5a8.png)

> normal mode

![2019-03-13 at 11 33](https://user-images.githubusercontent.com/6391776/54272330-c6cf0880-4583-11e9-8fc2-86bb9ff5a3d8.png)

> insert mode

**Comment by guillermooo** [comment]
The cursor appearance can only be controlled via ST native settings and themes. Six doesn't really do anything about it... I'll take a look, but I don't think there's much Six can do.

**Comment by PatTheMav** [comment]
I did some tests with the color schemes available by Sublime Text:

* Monokai - has blinking block cursor (light grey) in normal mode, bar cursor in insert mode, underscore in visual mode
* Breakers - block cursor is invisible in normal mode, bar cursor and underline both visible
* Celeste - blinking block cursor in normal mode, purple bar and underline cursor for insert and visual
* Mariana - block cursor is once again invisible, bar cursor and underline are orange
* Sixteen - again, block cursor is invisible, bar and underline are dark gray

As reported this started happening with Sublime 3.2 and makes normal mode more or less a guessing game of where the cursor currently might be.

**Comment by guillermooo** [comment]
@PatTheMav Thanks for that! Isn't this something that could be reported to the maintainers of the color schemes? (Even though I seem to remember that the new block caret appearance is controlled via theme settings?)

**Comment by PatTheMav** [comment]
@guillermooo Yeah it looks like affected themes have their block-caret colour not set, so it falls back to the line-Highlight colour, hence the appearance of invisibility.

It’s weird how this new feature in ST3 doesn’t seem to have been fully tested with all built-in themes, but those are the cards we’ve been dealt.
